aten/src/ATen/Context.cpp

```diff
@@ -263,7 +263,24 @@ void Context::setLinalgPreferredBackend(at::LinalgBackend b) {
   }
 }
 
-at::BlasBackend Context::blasPreferredBackend() const {
+at::BlasBackend Context::blasPreferredBackend() {
+#ifdef USE_ROCM
+  if (blas_preferred_backend == at::BlasBackend::Cublaslt) {
+    static const bool hipblaslt_unsupported = []() {
```

Comment:
Using `static` to ensure this variable is only defined once and `const` since it is assumed that the value of this variable will remain the same for every invocation since the machine configuration will be the same.